packages/manager/modules/iam/src/components/actionSelect/actionSelect.controller.js

```diff
@@ -257,6 +257,32 @@ export default class ActionSelectController {
     this.onModelChanged();
   }
 
+  /**
+   * Called back when the search query changes
+   * @param {string} actionTree
+   * @param {object} event
+   */
+  /* eslint-disable class-methods-use-this */
```

Comment:
**nitpick:** Dont disable ESLint rules. Declare the method as static and use `Controller.FunctionName` to call it.